Description: Keep kconf/zconf.{lex,tab}.c during "make clean"
Author: Anthony Wong <anthony.wong@canonical.com
Forwarded: https://www.spinics.net/lists/backports/msg04856.html
Last-Update: 2019-06-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile.real
+++ b/Makefile.real
@@ -127,7 +127,10 @@
 	@echo Reboot.
 	@echo
 
+EXCLUDE_CLEAN := kconf/zconf.lex.c kconf/zconf.tab.c
 .PHONY: clean
 clean:
+	@$(foreach f, $(EXCLUDE_CLEAN), mv $(f) $(f).orig;)
 	@$(MAKE) -f Makefile.build clean
 	@$(MAKE) -C kconf clean
+	@$(foreach f, $(EXCLUDE_CLEAN), mv $(f).orig $(f);)
